tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -270,7 +270,7 @@ jobs:
         inputs:
           command: 'custom'
           workingDir: ${{ parameters.testWorkspace }}/node_modules/${{ parameters.testPackage }}
-          customCommand: 'run ${{ parameters.testCommand }} -- ${{ variant.flags }}'
+          customCommand: 'run ${{ parameters.testCommand }} -- ${{ variant.flags }} --use-openssl-ca'
```

Comment:
Might be worth adding a brief comment here explaining why we need the flag, for future reference.

Reply:
Fair, adding it now. I expect it won't be necessary anymore once we're back on public certs, but doesn't hurt in the meantime.